### PR TITLE
refactor: destructure the assignment of `version`

### LINKED ISSUE #368: index.js needs to be rebuilt
Related #369

### LINKED ISSUE #369: Changes to package.json require the project to be rebuilt
PR #365 introduced a change that resulted in package.json being included in the final build. As a result, every change to package.json results in the project needing to be rebuilt. This includes trivial changes such as dependency bumps (for an example, see the now-merged #362).

### PR DESCRIPTION
**Pull request by RA80533**
Fixes #368
Fixes #369

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -7311,7 +7311,7 @@ module.exports = eval("require")("encoding");
...
+module.exports = {"i8":"1.5.2"};
```
Comment (RA80533) [inline dist/index.js:7314]: package.json was previously embedded in its entirety on [line 7314](https://github.com/codecov/codecov-action/blob/4463da3901eda7a7e4c92f2ada425a21369ada94/dist/index.js#L7314).

**Comment by RA80533** [comment]
@thomasrockhu, I think the additional commits here should be moved to a new PR.

Take a look at #376 for a starting point—the binary works as expected there.

**Comment by thomasrockhu** [comment]
@RA80533 agreed, I'll remove mine, I figured it would be an easier fix

**Comment by RA80533** [comment]
@thomasrockhu For future reference, use the built-in revert functionality of Git to undo commits to a branch instead of doing a forced push. Forced pushes are destructive to the commit log of the branch being pushed to; in this case, GitHub no longer contains the original history of the PR prior to the changes you made due to the forced push.

**Comment by RA80533** [comment]
I opened #394 to restrict instances of this from occurring in the future. This is a fairly common mistake for a lot of developers so it would likely be of great benefit to the project.

**Comment by RA80533** [comment]
This PR is superseded by #397.

**Pull request closed without merging**